.gitignore

```
@@ -59,3 +59,8 @@ temp/
 ._*
 .Trashes
 .svn
+
+############
+## Vendor Libraries
+############
+modules/images/image-loading-optimization/web-vitals/
```

I was thinking that we would copy the `dist/modules` directory, and if we do this then the version would have to be included as part of the directory name since adding a `ver` query arg to the initially-imported module wouldn't cache-bust the other modules it depends on. 

```suggestion
modules/images/image-loading-optimization/web-vitals-*/
```

Nevertheless, after re-looking at the `dist/modules` I see that they are not minified. Therefore, it probably is just better to import the standard build as you are doing in this PR.

---

Marking as resolved since updated the implementation.